CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Next
 
+## 3.25.0 – 2025-04-29
```

Comment:
similar to https://github.com/PostHog/posthog-ios/pull/345/files#r2065547767
thats why we have the 'Next'